framework-cloud/framework-openshift-operator/src/main/java/org/kie/cloud/openshift/operator/scenario/WorkbenchRuntimeSmartRouterImmutableKieServerWithDatabaseScenarioImpl.java

```diff
@@ -0,0 +1,193 @@
+
+            Auth auth = new Auth();
+            auth.setSso(sso);
+            kieApp.getSpec().setAuth(auth);
+        }
+
+        registerCustomTrustedSecret(kieApp.getSpec().getObjects().getConsole());
+        for (Server server : kieApp.getSpec().getObjects().getServers()) {
+            registerCustomTrustedSecret(server);
+        }
+
+        // deploy application
+        getKieAppClient().create(kieApp);
+
+        workbenchRuntimeDeployment = new WorkbenchRuntimeOperatorDeployment(project, getKieAppClient());
+        workbenchRuntimeDeployment.setUsername(DeploymentConstants.getWorkbenchUser());
+        workbenchRuntimeDeployment.setPassword(DeploymentConstants.getWorkbenchPassword());
+
+        smartRouterDeployment = new SmartRouterOperatorDeployment(project, getKieAppClient());
+
+        kieServerDeployment = new KieServerOperatorDeployment(project, getKieAppClient());
+        kieServerDeployment.setUsername(DeploymentConstants.getKieServerUser());
+        kieServerDeployment.setPassword(DeploymentConstants.getKieServerPassword());
+
+        databaseDeployment = new DatabaseDeploymentImpl(project);
+
+        logger.info("Waiting until all services are created.");
+        try {
+            new SimpleWaiter(() -> workbenchRuntimeDeployment.isReady()).reason("Waiting for Workbench service to be created.").timeout(TimeUnit.MINUTES, 1).waitFor();
+            new SimpleWaiter(() -> smartRouterDeployment.isReady()).reason("Waiting for Smart router service to be created.").timeout(TimeUnit.MINUTES, 1).waitFor();
+            new SimpleWaiter(() -> kieServerDeployment.isReady()).reason("Waiting for Kie server service to be created.").timeout(TimeUnit.MINUTES, 1).waitFor();
+            new SimpleWaiter(() -> databaseDeployment.isReady()).reason("Waiting for Database service to be created.").timeout(TimeUnit.MINUTES, 1).waitFor();
+        } catch (WaiterException e) {
+            throw new RuntimeException("Timeout while deploying application.", e);
+        }
+
+        logger.info("Waiting for Workbench deployment to become ready.");
+        workbenchRuntimeDeployment.waitForScale();
+
+        logger.info("Waiting for Smart router deployment to become ready.");
+        smartRouterDeployment.waitForScale();
+
+        logger.info("Waiting for Kie server deployment to become ready.");
+        kieServerDeployment.waitForScale();
+
+        logger.info("Waiting for Database deployment to become ready.");
+        databaseDeployment.waitForScale();
+
+        logger.info("Waiting for Kie server and Smart router to register itself to the Workbench.");
+        KieServerControllerClientProvider.waitForServerTemplateCreation(workbenchRuntimeDeployment, 2);
+
+        logNodeNameOfAllInstances();
+
+        // Used to track persistent volume content due to issues with volume cleanup
+        storeProjectInfoToPersistentVolume(workbenchRuntimeDeployment, "/opt/eap/standalone/data/kie");
```

Comment:
PV should be /opt/kie/data
```suggestion
        storeProjectInfoToPersistentVolume(workbenchRuntimeDeployment, "/opt/kie/data");
```